src/main/java/de/retest/recheck/cli/subcommands/Ignore.java

```diff
@@ -47,6 +61,81 @@ public void run() {
 				}
 			}
 		}
+		if ( inputValidation( all, testReport, list ) ) {
+			try {
+				final ReplayResult replayResult = ReplayResultProvider.load( testReport );
+
+				if ( !replayResult.containsChanges() ) {
+					logger.warn( "The test report has no differences." );
+					return;
+				}
+
+				try {
+					final LoadShouldIgnoreWorker loadShouldIgnoreWorker =
+							new LoadShouldIgnoreWorker( NopCounter.getInstance(), applier -> ignoreApplier = applier );
+					ignoreApplier = loadShouldIgnoreWorker.load();
+					logger.info( "The recheck ignore file loaded successfully." );
+				} catch ( final Exception e ) {
+					logger.error( "Could not load the recheck ignore file.", e );
+				}
+
+				ReplayResultProvider.print( replayResult, testReport );
+
+				if ( allDifferencesAreBlacklisted( replayResult ) ) {
+					logger.warn( "All differences in the given test report are already ignored." );
+					return;
+				} else {
+					logger.warn( "The recheck ignore file is being updated." );
```

Comment:
Should this be treated as a warning? As this process should be pretty fast, I'd either inform the user before **XOR** after saving the ignore.